precommit/mirsg-hooks.yaml

```
@@ -68,6 +68,7 @@ repos:
       - id: check-executables-have-shebangs
       - id: check-merge-conflict
       - id: check-shebang-scripts-are-executable
+        exclude: \.j2$
```

Do we need the `\` here?
```suggestion
        exclude: .j2$
```